src/components/HamburgerMenu.native.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { TouchableOpacity } from 'react-native'
 import Svg, { Path } from 'react-native-svg'
+import { useDrawerLayoutContext, IDrawerLayoutContext } from '../context/DrawerLayoutContext.native'
 
 //  const tutorialLocal = localStorage['Settings/Tutorial'] !== 'Off'
 
 /** An options menu with three little bars that looks like a hamburger. */
 const HamburgerMenu = ({ fill = '#fff', size = 45 }) => {
+  const { openDrawer } = useDrawerLayoutContext() as IDrawerLayoutContext
```

Comment:
is this state currently in redux? Is Raine ok for us to use context instead?

Reply:
Since we don't currently use context in the app, I would need to understand the advantages and disadvantages vs Redux state in order to make a decision.

There is some precedent for such a transition. We are gradually switching to `useSelector` from `mapStateToProps`.

Reply:
using redux now.